spec/spec_helper.rb

```diff
@@ -8,4 +8,9 @@
 
 RSpec.configure do |config|
   config.verbose_retry = true
+  config.display_try_failure_messages = true
```

Comment:
I tried, but RSpec didn't like it. Apparently you can't change the configuration after tests have already started running.